docs/multi-stage-query/api.md

```diff
@@ -102,7 +102,7 @@ headers = {
   'Content-Type': 'application/json'
 }
 
-response = requests.request("POST", url, headers=headers, data=payload)
+response = requests.post(url, headers=headers, data=payload, auth=(<username>, <password>))
```

Comment:
can we use python variables instead of bracketed variables to indicate username/password?
```suggestion
username = HTTP_AUTH_USER
password = HTTP_AUTH_PASSWORD
response = requests.post(url, headers=headers, data=payload, auth=(username, password))
```